src/package.json

```
@@ -43,6 +43,8 @@
     "node-fetch": "2.6.1",
     "xml-js": "1.6.11",
     "run-script-os": "1.1.3",
-    "node-watch": "0.7.0"
-  }
+    "node-watch": "0.7.0",
+    "rainbow-code": "^2.1.7"
+  },
+  "dependencies": {}
```

Not sure why this is needed?

```suggestion
  }
```